.drone.yml

```diff
@@ -246,34 +246,34 @@ pipeline:
       event: tag
       status: success
 
-  publish-vic-machine-server-dev:
-    image: plugins/gcr
+  vic-machine-server-tags:
+    image: '${TEST_BUILD_IMAGE=gcr.io/eminent-nation-87317/vic-integration-test:1.48}'
     pull: true
-    repo: eminent-nation-87317/vic-machine-server
-    dockerfile: cmd/vic-machine-server/Dockerfile
-    custom_dns: 10.142.7.21, 10.142.7.22, 10.166.17.90
-    secrets: [ token ]
-    tags:
-      - dev
+    environment:
+      BIN: bin
+      SHELL: /bin/bash
+    commands:
+      - 'echo -n "$(git describe --tags --abbrev=0)-${DRONE_BUILD_NUMBER}-$(git rev-parse --short=8 HEAD)" > .tags;'
+      - 'if [ "${DRONE_BUILD_EVENT}" = "push" -a "${DRONE_BRANCH}" = "master" ];     then echo ",dev" >> .tags; fi'
+      - 'if [ "${DRONE_BUILD_EVENT}" = "tag"  -a -n "$(echo "${DRONE_BRANCH}" | grep "releases/")" ]; then echo ",$(git describe --tags --abbrev=0),latest" >> .tags; fi'
+      - 'cat .tags'
     when:
       repo: vmware/vic
-      event: push
+      event: [push, tag]
       branch: [master, 'releases/*']
       status: success
 
-  publish-vic-machine-server-releases:
+  vic-machine-server-publish:
     image: plugins/gcr
```

Comment:
Did you test that `plugins/gcr` picks up tags from `.tags`? I'm not sure if this will work.

Reply:
Yes, see the script output above. You can also reference this: https://github.com/drone-plugins/drone-docker/blob/master/cmd/drone-docker/main.go#L122-L128

Reply:
Awesome!